tl.tl

```diff
@@ -3370,6 +3370,7 @@ local standard_library: {string:Type} = {
       a_type { typename = "function", args = {}, rets = { ALPHA, BETA } },
    } },
    ["pcall"] = a_type { typename = "function", args = { VARARG_ANY }, rets = { BOOLEAN, ANY } },
+   ["xpcall"] = a_type { typename = "function", args = { VARARG_ANY }, rets = { BOOLEAN, ANY } },
```

Comment:
on second thought, this should probably be

`args = { FUNCTION, FUNCTION, VARARG_ANY }`

and pcall`s should be `args = { FUNCTION, VARARG_ANY }` :thinking:

Reply:
i think so :)